quesma/processors/ab_test_processor.go

```diff
@@ -90,14 +86,11 @@ func (p *ABTestProcessor) Handle(metadata map[string]interface{}, message ...any
 		if len(messages) == 2 {
 			equal, diff := p.compare(string(messages[0]), string(messages[1]))
 			if equal {
-				resp = append(resp, []byte("ABTestProcessor processor: Responses are equal\n\n")...)
-				resp = append(resp, []byte("\n")...)
-				resp = append(resp, []byte(diff)...)
+				resp = append(resp, []byte("ABTestProcessor processor: Responses are equal\n")...)
 
 			} else {
-				resp = append(resp, []byte("ABTestProcessor processor: Responses are not equal\n\n")...)
-				resp = append(resp, []byte("\n")...)
-				resp = append(resp, []byte(diff)...)
+				resp = append(resp, []byte("ABTestProcessor processor: Responses are not equal\n")...)
+				fmt.Println(diff)
```

Comment:
Do you want to keep this diff?

Reply:
That was my plan, but after consideration I think it's not needed